bufferflow_timedraw.go

```diff
@@ -8,40 +8,55 @@ import (
 )
 
 type BufferflowTimedRaw struct {
-	Name   string
-	Port   string
-	Output chan []byte
-	Input  chan string
-	ticker *time.Ticker
+	port              string
+	output            chan []byte
```

Comment:
`output` type could be changed to `chan<- []byte`, same for other `Buffer`s, so it's clearer that a buffer is only meant to send to it.

Also in `NewBufferflowTimedRaw` args and the functions that create the other `Buffer`s.